src/src/cpp/environments/classic_control/ClassicControlEnv.h

```diff
@@ -5,54 +5,55 @@
 #include <GL/glut.h>
 #include <TaskEnv.h>
 
-using namespace std;
-
 class ClassicControlEnv : public TaskEnv {
    public:
-    uniform_real_distribution<> disReset;
-    uniform_real_distribution<> disNoise;
-    vector<deque<double> > actionTrace;
+    std::uniform_real_distribution<> dis_reset;
+    std::uniform_real_distribution<> dis_noise;
+    std::vector<std::deque<double>> action_trace;
+
     ClassicControlEnv() {
-        actionTrace.reserve(3);
-        actionTrace.resize(3);
+        action_trace.reserve(3);
+        action_trace.resize(3);
         for (size_t i = 0; i < 200; i++) {
-            actionTrace[0].push_back(0);
-            actionTrace[1].push_back(0);
-            actionTrace[2].push_back(0);
+            action_trace[0].push_back(0);
+            action_trace[1].push_back(0);
+            action_trace[2].push_back(0);
         }
-        disNoise = uniform_real_distribution<>(-M_PI, M_PI);
+        dis_noise = std::uniform_real_distribution<>(-M_PI, M_PI);
     }
+
     ~ClassicControlEnv() {}
 
-    double bound(double x, double m, double M) { return min(max(x, m), M); }
-    virtual void display_function(int, int, double) {};
+    double Bound(double x, double m, double M) {
+        return std::min(std::max(x, m), M);
+    }
+    virtual void DisplayFunction(int, int, double){};
 
-    /**************************************************************************/
-    void saveScreenshotToFile(std::string filename, int windowWidth,
-                              int windowHeight) {
-        const int numberOfPixels = windowWidth * windowHeight * 3;
-        unsigned char pixels[numberOfPixels];
+    void SaveScreenshotToFile(std::string filename, int window_width,
```

Comment:
this method (maybe some other ones too) are used in [here](https://github.com/TPGEngine/tpg/blob/main/src/src/cpp/experiments/evaluators_control.h#L37), so maybe you'll have to make changes there too.